profile/README.md

```diff
@@ -2,7 +2,7 @@
 
 # Sovereign Cloud Stack
 
-[Sovereign Cloud Stack (SCS)](https://scs.community/) is a network of existing and future providers of standardized sovereign cloud and container infrastructure to join forces in defining, implementing and operating a fully open, federated, compatible platform. By joining forces, the platform can support a healthy ecosystem for service and application developers and strengthen the digitalization efforts in Europe without creating large risks to lose control over the technology and the data.
+The [OSB Alliance](https://osb-alliance.de/) is a network of existing and future providers that develops the [Sovereign Cloud Stack (SCS)](https://scs.community/) of standardized sovereign cloud and container infrastructure to join forces in defining, implementing and operating a fully open, federated, compatible platform. By joining forces, the platform can support a healthy ecosystem for service and application developers and strengthen the digitalization efforts in Europe without creating large risks to lose control over the technology and the data.
```

Comment:
The OSBA is a non-profit organization with more than 200 members ranging from Open Source industry, Open source ISVs, academia, personal memberships. 
While I agree that the README.md can probably be improved, it likely needs different wording.